src/Tizen.NUI/src/internal/WebView/WebViewContextMenuItemSelectedEventArgs.cs

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Event arguments that passed via the WebView.ContextMenuItemSelected.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebViewContextMenuItemSelectedEventArgs : EventArgs
+    {
+        internal WebViewContextMenuItemSelectedEventArgs(WebContextMenuItem item)
+        {
+            ContextMenuItem = item;
+        }
+
+        /// <summary>
+        /// The context menu.
```

Comment:
I think "The selected context menu item" is better here.

Reply:
Done.